src/models/game_session.ts

```diff
@@ -190,18 +210,15 @@ export default class GameSession {
             this.lastPlayedSongsQueue.shift();
         }
 
-        this.sessionInitialized = true;
         let randomSong: QueriedSong;
         try {
             randomSong = await selectRandomSong(guildPreference, this.lastPlayedSongsQueue);
             if (randomSong === null) {
-                this.sessionInitialized = false;
```

Comment:
why was this removed?

Reply:
My mistake, I wasn't sure where to set sessionInitialized back to false. Does it make sense to move it directly into the endSession method?

Reply:
nvm, this might be fine. just make sure typing `,play` multiple times in succession doesn't have any issues

Reply:
I just tested ,play multiple times quickly and it behaved as expected (only one start message, the rest as "Game in session")